interfaces/acados_template/acados_template/c_templates_tera/Makefile.in

```diff
@@ -31,162 +31,253 @@
 # POSSIBILITY OF SUCH DAMAGE.;
 #
 
+{% if solver_options.qp_solver %}
+	{%- set qp_solver = solver_options.qp_solver %}
+{% else %}
+	{%- set qp_solver = "FULL_CONDENSING_HPIPM" %}
+{% endif %}
+
+{% if solver_options.hessian_approx %}
+	{%- set hessian_approx = solver_options.hessian_approx %}
+{% else %}
+	{%- set hessian_approx = "GAUSS_NEWTON" %}
+{% endif %}
+
+{% if constraints.constr_type %}
+	{%- set constr_type = constraints.constr_type %}
+{% else %}
+	{%- set constr_type = "BLA" %}
```

Comment:
what is the meaning of 'BLA'?

Reply:
This is just to use the same Makefile for the standalone integrator.
To explicitly not render the constraint part below.
I replace `BLA` with `NONE`